### PR TITLE
Add LCM algorithm to math library

### PR DESCRIPTION
**Pull request by sekomer**
Add LCM algorithm to math library

## Description
Least common multiplier method added to `u64` and `u128` math libraries, along with tests.

## Type of Change
- [x] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [x] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Note
* changes tested with `cargo test` command.
* docs regenerated using the `cargo build -p aptos-cached-packages` command.

### CODE REVIEW

**Comment by sekomer** [comment]
hey @junkil-park, can you check it?

File: aptos-move/framework/aptos-stdlib/sources/math64.move [hunk 1]
```diff
@@ -37,6 +37,15 @@ module aptos_std::math64 {
         large
     }
 
+    /// Returns least common multiple of `a` & `b`.
+    public inline fun lcm(a: u64, b: u64): u64 {
+        if (a == 0 || b == 0) {
+            0
```
Comment (junkil-park) [inline aptos-move/framework/aptos-stdlib/sources/math64.move:43]: Should it return 0 or abort when a == b == 0?

Comment (sekomer) [reply aptos-move/framework/aptos-stdlib/sources/math64.move:43]: according to [wikipedia](https://en.wikipedia.org/wiki/Least_common_multiple#:~:text=Using%20the%20greatest%20common%20divisor,-The%20least%20common&text=However%2C%20if%20both%20a%20and,considered%20as%20a%20special%20case.), lcm(0, 0) is a special case, and should be considered as 0.

some popular languages such as [python](https://github.com/python/cpython/blob/0dcbc8385322ff51f7fc3e586027d880275df4fa/Modules/mathmodule.c#L815) also accepts `lcm(0, 0) = 0`

**Comment by sekomer** [comment]
hi @junkil-park, is there anything else

File: aptos-move/framework/aptos-stdlib/sources/math64.move (hunk 1 above)
Comment (sekomer) [reply aptos-move/framework/aptos-stdlib/sources/math64.move:43]: hi @junkil-park, is there anything else?

Comment (sekomer) [reply aptos-move/framework/aptos-stdlib/sources/math64.move:43]: @movekevin @wrwg @davidiw @junkil-park

**Review by junkil-park — APPROVED** [review]

File: aptos-move/framework/aptos-stdlib/sources/math64.move (hunk 1 above)
Comment (junkil-park) [reply aptos-move/framework/aptos-stdlib/sources/math64.move:43]: The CI tests are running.

Comment (sekomer) [reply aptos-move/framework/aptos-stdlib/sources/math64.move:43]: I guess some tests are awaiting approval

Comment (sekomer) [reply aptos-move/framework/aptos-stdlib/sources/math64.move:43]: @junkil-park hi, any update?

Comment (junkil-park) [reply aptos-move/framework/aptos-stdlib/sources/math64.move:43]: The CI tests passed. Two approvals are required to merge PRs. I've already approved it, and it needs one more approval.

Comment (sekomer) [reply aptos-move/framework/aptos-stdlib/sources/math64.move:43]: @davidiw @movekevin @wrwg UP

Comment (sekomer) [reply aptos-move/framework/aptos-stdlib/sources/math64.move:43]: @junkil-park  @davidiw @movekevin @wrwg

I was planning to contribute Aptos more, but seems like its not even open source 🤣🤣🤣

**Comment by junkil-park** [comment]
@JohnChangUK, does this PR look good to you ?

**Pull request merged**